test/cql-pytest/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==6.2.5
+cassandra-driver==3.25.0
+pytest-repeat==0.9.1
```

Comment:
I want to understand what this means and why you think it's useful. What does "==" mean, that you *need* those specific versions? Because that's not true. We tried to make sure that the tests actually work on all reasonable versions of these tools. I use pytest 6.2.2 and it's working perfectly, no need for 6.2.5. For example.

As for Cassandra driver, we allow either the regular one (from Datastax), or from Scylla, and the version numbers mean different things for each. When we have specific tests that can't work on older drivers, these tests are skipped (see for example test_filtering.py, test_filtering_contiguous_nonmatching_partition_range) but that's not a reason someone can't run most tests with an older driver.

Is there a way we could say that we need the "cassandra" package, without saying anything about the version?

Reply:
Done